madgraph/iolibs/export_v4.py

```diff
@@ -7640,6 +7652,7 @@ def create_couplings_main(self, nb_def_by_file=25):
                                     include \'mp_coupl.inc\'
                             """%self.mp_real_format)
             fsock.writelines("""include \'input.inc\'
+                                include \'../vector.inc\'
                                 include \'coupl.inc\'
```

Comment:
same (again one which is less clear if we need it or not)

Reply:
same as above, vector.inc is needed by coupl.inc